src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -20,227 +129,2379 @@ const ActorInit En_Ossan_InitVars = {
+        osSyncPrintf(VT_RST);
+        __assert("0", "../z_en_oB1.c", 1284);
+        return;
+    }
+
+    if (func_80AC33B0(this, globalCtx, tmp) == 0) {
+        Actor_Kill(thisx);
+        osSyncPrintf(VT_COL(RED, WHITE));
+        // No reserve bank
+        osSyncPrintf("予備バンクが無いよ！！(%s)\n", D_80AC8914[this->actor.params]);
+        osSyncPrintf(VT_RST);
+        __assert("0", "../z_en_oB1.c", 1295);
+        return;
+    }
+
+    Actor_ProcessInitChain(thisx, sInitChain);
+    func_80AC2CA0(this, func_80AC7094);
+}
+
+void EnOssan_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnOssan* this = THIS;
+
+    SkelAnime_Free(&this->skelAnime, globalCtx);
+    Collider_DestroyCylinder(globalCtx, &this->collider);
+}
+
+void func_80AC3744(GlobalContext* globalCtx, EnOssan* this) {
+    s16 sp1E;
+    s16 sp1C;
+
+    func_8002F374(globalCtx, &this->unk_200[this->unk_252]->actor, &sp1E, &sp1C);
+    this->unk_230 = (f32)sp1E;
+    this->unk_234 = (f32)sp1C;
+}
+
+void func_80AC37A8(GlobalContext* globalCtx, EnOssan* this) {
+    Player* player = PLAYER;
+
+    // Convo over
+    osSyncPrintf(VT_FGCOL(YELLOW) "%s[%d]:★★★ 会話終了！！ ★★★" VT_RST "\n", "../z_en_oB1.c", 1337);
+    YREG(31) = 0;
+    func_8002F194(&this->actor, globalCtx);
+    globalCtx->msgCtx.msgMode = 0x36;
+    globalCtx->msgCtx.unk_E3E7 = 4;
+    player->stateFlags2 &= ~0x20000000;
+    func_800BC490(globalCtx, 1);
+    Interface_ChangeAlpha(50);
+    this->unk_251 = 0;
+    this->unk_288 = 0;
+    this->unk_2C0 = 0;
+    func_80AC3350(this, globalCtx, 0.0f);
+    this->actor.textId = func_80AC6F2C(this);
+    this->unk_1FC = 0;
+}
+
+s32 func_80AC3884(EnOssan* this, GlobalContext* globalCtx, Input* cont1) {
+    if (CHECK_PAD(cont1->press, B_BUTTON)) {
+        func_80AC37A8(globalCtx, this);
+        return 1;
+    } else {
+        return 0;
+    }
+}
+
+s32 func_80AC38C8(EnOssan* this, GlobalContext* globalCtx, Input* cont1) {
+    if (CHECK_PAD(cont1->press, B_BUTTON)) {
+        this->unk_1FC = this->unk_1FE;
+        func_8010B720(globalCtx, this->unk_200[this->unk_252]->actor.textId);
+        return 1;
+    } else {
+        return 0;
+    }
+}
+
+void func_80AC3928(GlobalContext* globalCtx, EnOssan* this, u8 arg2) {
+    YREG(31) = 1;
+    this->unk_1F0 = 0;
+    this->unk_1EE = this->unk_1F0;
+    Interface_SetDoAction(globalCtx, 0x10);
+    func_80AC3350(this, globalCtx, 0.0f);
+    if (!arg2) {
+        this->unk_1FC = 1;
+    } else {
+        func_80AC39AC(globalCtx, this);
+    }
+}
+
+void func_80AC39AC(GlobalContext* globalCtx, EnOssan* this) {
+    this->unk_1FC = 2;
+    if (this->actor.params == 0xA) {
+        if ((gSaveContext.itemGetInf[3] & 0x100) && (gSaveContext.itemGetInf[3] & 0x200) &&
+            (gSaveContext.itemGetInf[3] & 0x400) && (gSaveContext.itemGetInf[3] & 0x800)) {
+            // Choose a mask with left or right
+            // on [Control Stick].
+            // Ask about Mask of Truth
+            // Don't borrow a mask 
```

Comment:
I say remove any text comments. I know there was a miscommunication about whether or not we want these in, but its really not worth clogging up the code with this stuff